Assets/BossRoom/Scripts/Server/Net/ServerGameNetPortal.cs

```diff
@@ -312,25 +278,34 @@ private IEnumerator WaitToDisconnectClient(ulong clientId, ConnectStatus reason)
         /// This method will summarily remove a player connection, as well as its controlled object.
         /// </summary>
         /// <param name="clientId">the ID of the client to boot.</param>
-        public void BootClient(ulong clientId)
+        void BootClient(ulong clientId)
         {
-            var netObj = MLAPI.Spawning.NetworkSpawnManager.GetPlayerNetworkObject(clientId);
+            var netObj = NetworkSpawnManager.GetPlayerNetworkObject(clientId);
             if( netObj )
             {
                 //TODO-FIXME:MLAPI Issue #795. Should not need to explicitly despawn player objects.
                 netObj.Despawn(true);
             }
-            m_Portal.NetManager.DisconnectClient(clientId);
+            NetworkManager.Singleton.DisconnectClient(clientId);
         }
 
         /// <summary>
         /// Called after the server is created-  This is primarily meant for the host server to clean up or handle/set state as its starting up
         /// </summary>
-        private void ServerStartedHandler()
+        void ServerStartedHandler()
         {
-            m_ClientData.Add("host_guid", new PlayerData(m_Portal.PlayerName, m_Portal.NetManager.LocalClientId));
-            m_ClientIDToGuid.Add(m_Portal.NetManager.LocalClientId, "host_guid");
-        }
+            var localClientID = NetworkManager.Singleton.LocalClientId;
 
+            m_ClientData.Add("host_guid", new PlayerData(m_Portal.PlayerName, localClientID));
+            m_ClientIDToGuid.Add(localClientID, "host_guid");
+
+            var newPlayer = Instantiate(m_PlayerPrefab, Vector3.zero, Quaternion.identity, null);

```

Comment:
let's make sure we can't use MLAPI's default player spawn flow before spawning ourselves. And please document in code anything you find